src/three/raycastTraverse.js

```diff
@@ -87,45 +87,47 @@ export function raycastTraverseFirstHit( root, group, activeTiles, raycaster ) {
 			_mat.multiply( obbMat ).invert();
 			_ray.copy( raycaster.ray );
 			_ray.applyMatrix4( _mat );
-			if ( _ray.intersectBox( boundingBox, _vec ) ) {
+			if ( ! _ray.intersectBox( boundingBox, _vec ) ) {
 
-				// account for tile scale
-				_vec2.setFromMatrixScale( _mat );
-				const invScale = _vec2.x;
+				continue;
 
-				if ( Math.abs( Math.max( _vec2.x - _vec2.y, _vec2.x - _vec2.z ) ) > 1e-6 ) {
+			}
 
-					console.warn( 'ThreeTilesRenderer : Non uniform scale used for tile which may cause issues when raycasting.' );
+		} else {
 
-				}
+			_mat.invert();
```

Comment:
The logic changes in this raycast first hit function don't look right to me. If we want to accommodate bounding spheres we need to try to intersect both the sphere and the obb. If either ray misses we should skip that child and not add it to the array to check later. Otherwise we need to take the shortest distance between the two hits and save it in the array so we can use it for sorting and check the nearest bound first.

It might be best to make these raycast changes in a separate PR so they're easier to follow.

Reply:
I will remove the raycast changes from this PR to keep is simple and submit the raycast changes as a new PR.